actioncable/README.md

```diff
@@ -134,6 +134,7 @@ class AppearanceChannel < ApplicationCable::Channel
   end
 end
 ```
+NOTE: You must implement `appear`, `disappear`, and `away` methods for `current_user`
```

Comment:
Two small grammar items:

- add a `the` after implement
- add a period at the end of the sentence